boards/arm/stm32h745i_disco/stm32h745i_disco_m7.dts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2022 Systerel (https://www.systerel.fr)
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/h7/stm32h745Xi_m7.dtsi>
+#include <st/h7/stm32h745zitx-pinctrl.dtsi>
+#include "stm32h745i_disco.dtsi"
+	
+/*
+ * WARNING:
+ * Possible pin conflicts: The pins PA2 and PB13 may conflict on selection of
+ * ETH_STM32_HAL, since they are used in ST Zio or ST morpho connectors. To
+ * avoid conflicting states the jumpers JP6 and JP7 must be in ON state.
+ */
+
+/ {
+	model = "STMicroelectronics STM32H745I-DISCO board";
+	compatible = "st,stm32h745i-disco";
+
+	/* HW resources belonging to CM7 */
+	chosen {
+		zephyr,console = &usart3;
+		zephyr,shell-uart = &usart3;
+		zephyr,dtcm = &dtcm;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+
+		red_pwm_led: red_pwm_led {
+			pwms = <&pwm12 1 4 PWM_POLARITY_NORMAL>;
+			label = "User LD3 - PWM12";
+		};
+	};
+
+	aliases {
+		led0 = &green_led;
+		pwm-led0 = &red_pwm_led;
+		sw0 = &user_button;
+	};
+};
+
+&clk_hse {
+	hse-bypass;
+	clock-frequency = <DT_FREQ_M(8)>; /* STLink 8MHz clock */
```

Comment:
According to user manual 
```suggestion
	clock-frequency = <DT_FREQ_M(25)>; /* STLink 8MHz clock */
```

Reply:
clock-frequency = <DT_FREQ_M(25)>; /* STLink 25MHz clock */